src/datetime.js

```diff
@@ -1763,7 +1763,7 @@ export default class DateTime {
    */
   toRelative(options = {}) {
     if (!this.isValid) return null;
-    const base = options.base || DateTime.local(),
+    const base = options.base || DateTime.local().setZone(this.zone.name),
```

Comment:
I'd prefer this was `DateTime.fromObject({ zone: this.zone })`

Reply:
@icambron  Thanks for the suggestion